README.md

```diff
@@ -185,6 +185,11 @@ The username will for login will be `rstudio` and the password will be whatever
 
 If you are a new user, you may find [these instructions](https://github.com/AlexsLemonade/RNA-Seq-Exercises/blob/master/docker-pull.md) for a setting up a different Docker container or [this guide](https://www.andrewheiss.com/blog/2017/04/27/super-basic-practical-guide-to-docker-and-rstudio/) from Andrew Heiss helpful.
 
+We have supplied an RStudio project (`OpenPBTA-analysis.Rproj`) file at the base of the project to aid in organization and encourage reproducible defaults for analysis.
```

Comment:
Under the `Development in the Project Docker Container` header seems like an odd spot for this information in my opinion because I believe these are the defaults = this matters more for local development within RStudio _outside_ of the Docker container.

Reply:
Agreed, but that is the only place RStudio was mentioned. (There is utility for Docker users too, but not so much the reproducibility part.) 

Ideas on where it should go?

Reply:
A section under how to add an analysis called `Local development` and add an `RStudio` subheader?

Reply:
Done... see what you think?

Reply:
I like it 👍